page/config/manual/languages.yml

```diff
@@ -5,4 +5,5 @@ de:
   languageName: Deutsch
   weight: 2
   title: 'Contao Handbuch'
-  documentation_label: Handbuch
+  params:
+    documentation_label: Handbuch
```

Comment:
Is that a change necessary within the new Hugo version?

Reply:
> Is that a change necessary within the new Hugo version?

This change is not necessary, but recommended since hugo version v0.112.0:

Excerpt from the offiicial [hugo docs](https://gohugo.io/content-management/multilingual/#changes-in-hugo-01120):

```
Adding custom parameters to the top level language configuration is deprecated.
Define custom parameters within languages.xx.params.
```